--- node_modules/requirejs/require.js	2012-01-02 22:39:13.000000000 +0000
+++ node_modules/requirejs/require.js-normalizeFix	2012-01-02 22:40:42.000000000 +0000
@@ -319,7 +319,7 @@
                     url = urlMap[normalizedName];
                     if (!url) {
                         //Calculate url for the module, if it has a name.
-                        url = context.nameToUrl(normalizedName, null, parentModuleMap);
+                        url = context.nameToUrl(name, null, parentModuleMap);
 
                         //Store the URL mapping for later.
                         urlMap[normalizedName] = url;
